clients/privacy-center/scripts/validate-config.js

```diff
@@ -9,7 +9,14 @@ const configIsValid = (
 ) => {
   // Cannot currently have more than one consent be executable
   if (config.consent) {
-    const executables = config.consent.consentOptions.filter(
+    let options = [];
+    const isV1Consent = typeof config.consent?.page === "undefined";
+    if (isV1Consent) {
+      options = config.consent?.consentOptions
+    } else {
+      options = config.consent?.page.consentOptions
+    }
```

Comment:
would typically use a ternary for something like this

```suggestion
    options = isV1Consent ? config.consent?.consentOptions : config.consent?.page.consentOptions;
```

Reply:
and then the `let options = []` might not be needed since all cases would be covered by the if/else